components/viz/test/run_all_unittests.cc

```diff
@@ -18,7 +18,7 @@ static int InitAndRunAllTests(int argc, char** argv) {
 }
 
 // For the Starboard OS define SbEventHandle as the entry point
-SB_EXPORT STARBOARD_WRAP_SIMPLE_MAIN(InitAndRunAllTests);
+SB_EXPORT STARBOARD_WRAP_SIMPLE_MAIN(InitAndRunAllTests)
```

Comment:
Nitpick, same comment elsewhere: I think those macros should be written to accept semicolons without generating a warning.